src/Raven.Server/Documents/Sharding/Handlers/Processors/Revisions/ShardedRevisionsHandlerProcessorForGetRevisions.cs

```diff
@@ -75,7 +75,7 @@ protected override async ValueTask GetRevisionsAsync(TransactionOperationContext
             int shardNumber = RequestHandler.DatabaseContext.GetShardNumberFor(context, id);
             var result = await RequestHandler.ExecuteSingleShardAsync(context, cmd, shardNumber, token);
 
-            string actualEtag = cmd.Etag;
```

Comment:
the actual issue is in the impl of NotModified I think?

Reply:
done